### DISCUSSION TITLE
Upgrade to 3.4.7 causes previously working code to hang

### DISCUSSION THREAD

**Discussion by alpha7designs** [ultrafunkamsterdam/undetected-chromedriver#1281] [General]
I had code that worked then one site implemented the a "Click and Hold" captcha.  I upgraded uc to see if that would help and now the code hangs on this line:

driver = uc.Chrome(options=chrome_options, version_main=112)

Can anyone help?

Thanks!

**Comment by Lolmc0587** [comment]
please give me the log

**Reply by alpha7designs** [reply]
Where do I find the log?
